openpype/plugins/publish/extract_review_slate.py

```diff
@@ -314,6 +308,41 @@ def process(self, instance):
                 slate_subprocess_cmd, shell=True, logger=self.log
             )
 
+            # Create slate with silent audio track
+            if input_audio:
+                # silent slate output path
+                slate_silent_path = slate_path.replace(".png", "Silent" + ext)
+                _remove_at_end.append(slate_silent_path)
+
+                slate_silent_args = [
+                    ffmpeg_path,
+                    "-i", slate_v_path,
+                    "-f", "lavfi", "-i",
+                    "anullsrc=r={}:cl={}:d={}".format(
+                        audio_sample_rate,
+                        audio_channel_layout,
+                        one_frame_duration
+                    ),
+                    "-c:v", "copy",
+                    "-c:a", audio_codec,
+                    "-map", "0:v",
+                    "-map", "1:a",
+                    "-shortest",
```

Comment:
There are missing format arguments.

Reply:
Maybe irrelevat comment if this is true https://github.com/pypeclub/OpenPype/pull/3162/files#r869999097